### ISSUE TITLE
Fail when using commands from inventory file

### ISSUE THREAD

**Issue by pmarcais** [hellt/cmdo#11]
It seems there is a regression when using commands from the inventory file.

```
 devices:
  sros:
    platform: nokia_sros_classic
    address:myrouter
    send-commands:
      - show version
      - show router interface
      - admin display-config
```

In this case cmds array has one empty entry in inventory.go.  I workarround the issue for now by testing cmds[0] but there maybe a beter way.
```
cmds := strings.Split(app.commands, "::")

	if cmds[0] != "" {
		for _, device := range i.Devices {
			device.SendCommands = cmds
		}
	}
```

**Comment by hellt** [comment]
thanks @pmarcais for reporting this
I will have a look

**Comment by hellt** [comment]
This should have been fixed in 0.5.1
